### ISSUE TITLE
Use pip install requirements instead of poetry

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#689]
**TL;DR**: Install our project within the Dockerfile by using `pip install .`.

**Narrative**
Currently we have to install poetry to install our project's libraries.  In order for poetry to be installed it has to install its own python libraries.  These libraries might (unlikely) conflict with our own library requirements.  Additionally, poetry is intended as a development tool, not a deployment tool.  

To avoid any possible issues, we should be deploying our project with `pip install .`, which would require a reorganization of our directory structure.  

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests
- [ ] Scoping
- [ ] Alternatives Considered
- [ ] Rico sign-off

**Implementation Notes**
This is possible, but messy.
